client/my-sites/plugins/marketplace/index.tsx

```diff
@@ -0,0 +1,42 @@
+export type MarketplaceThemeType = {
+	colors: { textColorLight: string; borderColorLight: string };
+	breakpoints: {
+		desktopUp?: string;
+		tabletUp?: string;
+		bigPhoneUp?: string;
+		smallPhoneUp?: string;
+		tabletDown: string;
+	};
+	weights: {
+		bold: string;
+		normal: string;
+	};
+	fonts: {
+		body: string;
+	};
+	fontSize: {
+		small: string;
+	};
+};
+
+const theme: MarketplaceThemeType = {
+	colors: {
+		borderColorLight: 'var(--studio-gray-0)',
+		textColorLight: 'var(--color-text)',
+	},
+	breakpoints: {
+		tabletDown: 'max-width: 782px',
+	},
+	weights: {
+		bold: '600',
+		normal: '400',
+	},
+	fonts: {
+		body:
+			'-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif',
```

Comment:
Do you plan to use this theme font property anywhere? The fonts should be inherited from the base Calypso styles instead of being specified in the theme.

Reply:
Nope will clean this up